test/stdlib/heapsort_r.c

```diff
@@ -59,7 +59,27 @@ static int cmp64(void* thunk, const void* a, const void* b)
 	return *ua < *ub ? -1 : *ua != *ub;
 }
 
+static int icmp_thunk_check(void* thunk, const void* a, const void* b)
+{
+	assert_ptr_equal(thunk, input_thunk);
+	return *(const int*)a - *(const int*)b;
+}
+
+static void heapsort_r_int_test_1(void** state)
```

Comment:
Naming feedback applies here - the name should reference the `thunk` aspect of the test in some way.